config/v1/types_node.go

```diff
@@ -63,6 +67,35 @@ const (
 	DefaultUpdateDefaultReaction WorkerLatencyProfileType = "Default"
 )
 
+var (
+	// DefaultNodeStatusUpdateFrequency refers to the "--node-status-update-frequency" of the kubelet in case of DefaultUpdateDefaultReaction WorkerLatencyProfile type
+	DefaultNodeStatusUpdateFrequency = metav1.Duration{Duration: 10 * time.Second}
```

Comment:
@soltysh @rphillips I was wondering if we should have `const` for increased safety of the profiles, just to make sure any code using cannot accidently change it.

Reply:
In case of cgroups mode, we are indeed using constants - https://github.com/openshift/api/blob/24043075985b3dc87190174241f3c4d0416b8ea2/config/v1/types_node.go#L46

Reply:
Other examples, 
https://github.com/openshift/api/blob/a6156965faae5ce117e3cd3735981a3fc0e27e27/networkoperator/v1/types_egressrouter.go#L138-L157

Reply:
The overall idea seems reasonable, my only suggestion would be to differentiate between `Duration` and `int` flags. For example, `--default-not-ready-toleration-seconds` expects just `int` value, whereas `--node-monitor-grace-period` expects `Duration`. In this case it would be good to define these flags so they match their expected value. 

For the "consts" usage I'd like a second opinion from @deads2k

Reply:
Addressed

Reply:
Consts are fine.